streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -352,7 +352,9 @@ public StreamTask createStreamTask(TaskId id, Collection<TopicPartition> partiti
 
 
         void waitUntilTasksUpdated() {
```

Comment:
> I'm assuming we are refactoring it in another JIRA / PR

@guozhangwang - Yes I left this function in this state as I plan to refactor and pull this up into TestUtils as part of KAFKA-3842